omero/users/cli/import.rst

```diff
@@ -22,17 +22,17 @@ Visit :doc:`installation` to install the |CLI|.
 Import command
 ^^^^^^^^^^^^^^
 
-.. program:: omero import
-
 To import a file :file:`image.tif`, use::
 
     $ bin/omero import image.tif
 
 Some of the options available to the import command are:
 
+.. program:: omero import
+
 .. option:: -h, --help
 
-Examples of options available to the import command,
+    Examples of options available to the import command,
 
 .. option:: -s SERVER, -p PORT, -U USERNAME, -g GROUPNAME
```

Comment:
Why are these not documented separately?

Reply:
I guess because lumping them together saves a lot of repetition given they are somewhat self-explanatory

Reply:
I suppose, but it does prevent them being cross-referenced.  Could we list them separately for consistency and to allow referencing?